content/en/tracing/other_telemetry/connect_logs_and_traces/opentelemetry.md

```diff
@@ -46,10 +46,12 @@ class CustomDatadogLogProcessor(object):
         # An example of adding datadog formatted trace context to logs
         # from: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/b53b9a012f76c4fc883c3c245fddc29142706d0d/exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/propagator.py#L122-L129 
         current_span = trace.get_current_span()
+        if not current_span.is_recording():
+            return eventDict
 
-        if current_span is not None:
-            event_dict['dd.trace_id'] = str(current_span.context.trace_id & 0xFFFFFFFFFFFFFFFF)
-            event_dict['dd.span_id'] = str(current_span.context.span_id)
+        if context := current_span.get_span_context():
```

Comment:
Feel free to move this syntax into something 2.x compatible. Or ping me and I'll do so.

Reply:
:wave: Thank you for the fix! 

We can target Python 3 but we do need to support Python 3.7, so we should avoid the walrus operator.

Reply:
```suggestion
        context = current_span.get_span_context() if current_span is not None else None
        if context is not None:
```
Shouldn't be required given the typing, but we'll be a little defensive.